packages/sui-studio/bin/sui-studio-generate.js

```diff
@@ -139,6 +141,25 @@ describe${context ? '.context.default' : ''}('${componentInPascal}', ${
 })
 `
 
+const componentTemplate = `// import PropTypes from 'prop-types'
+
+export default function ${componentInPascal}() {
+  return (
+    <div className="${prefix}-${componentInPascal}">
+      <h1>${componentInPascal}</h1>
+    </div>
+  )
+}
+
+${componentInPascal}.displayName = '${componentInPascal}'
+${componentInPascal}.propTypes = {}
+`
+
+const explicitImportTemplate = `import ${componentInPascal} from './${componentInPascal}.js'
+
+export default ${componentInPascal}
+`
```

Comment:
Maybe we can go with the shorter version that mixes the rexport of default and named exports.
```suggestion
const explicitImportTemplate = `export {default} from './${componentInPascal}.js'
`
```
Then if we want to export some named (for example: foo and bar) things inside the component would be much shorter:

`const explicitImportTemplate = `export {default, foo, bar} from './${componentInPascal}.js'`

Reply:
This is a proposal in stage 1, I checked and we use `@babel/plugin-proposal-export-default-from` to support it. If we're sure our infra will keep it, I'm ok with the change, but just want to double check 🙂